src/pyxplor/pyxplor.py

```diff
@@ -0,0 +1,52 @@
+def time_series_explorer(input_df: pd.DataFrame, 
+                         date_column: str, 
+                         value_column: str, 
+                         freq: str = 'D', 
+                         plot_type: str = 'line', 

```

Comment:
Based on decision of @phchen5 's comment below re: removing plot type:

```suggestion

```

Reply:
I agree!